util/scale_run.sh

```diff
@@ -0,0 +1,335 @@
+#!/bin/bash
+
+# Run scale tests for each Standard Ref Arch size on docs page
+#   Ref: https://puppet.com/docs/pe/latest/hardware_requirements.html
+
+##########
+# Gatling settings
+##########
+#
+# Change these values if needed prior to running the script.
+#
+
+# export PUPPET_GATLING_SCALE_TUNE_FORCE=true
+
+# AWS instance sizes to be tested on
+declare -a TRIAL_EC2_TYPES=("c5.large")
```

Comment:
Aren't we using m5.large for the trial instance type?

Reply:
:man_facepalming: yes. Updated.